scripts/start.mts

```diff
@@ -0,0 +1,178 @@
+import process from 'node:process';
+import child_process from 'node:child_process';
+import fs from 'node:fs/promises';
+import path from 'node:path';
+import timers from 'node:timers/promises';
+import { pipeline } from 'node:stream/promises';
+import { Transform } from 'node:stream';
+
+// Get raw arguments after the script name, skipping node and script path
+const args = process.argv.slice(2);
+
+// Check if help flag appears before any target (making it "our" help)
+const validTargets = ['desktop', 'sandbox', 'sync'];
+const firstTargetIndex = args.findIndex((arg) => validTargets.includes(arg));
+const helpIndex = args.findIndex((arg) => arg === '-h' || arg === '--help');
+
+const isOurHelp =
+  helpIndex !== -1 && (firstTargetIndex === -1 || helpIndex < firstTargetIndex);
+
+if (isOurHelp) {
+  console.log(
+    await fs.readFile(path.join(import.meta.dirname, 'start.md'), 'utf8')
+  );
+  process.exit(0);
+}
+
+// Parse positional arguments and group them by target
+const targets = {
+  desktop: { enabled: false, args: [] as string[] },
+  sandbox: { enabled: false, args: [] as string[] },
+  sync: { enabled: false, args: [] as string[] },
+};
+
+let currentTarget: keyof typeof targets | null = null;
+
+for (const arg of args) {
+  if (validTargets.includes(arg as keyof typeof targets)) {
+    // Switch to new target
+    currentTarget = arg as keyof typeof targets;
+    targets[currentTarget].enabled = true;
+  } else if (currentTarget) {
+    // Add argument to current target's args (including help flags destined for targets)
+    targets[currentTarget].args.push(arg);
+  }
+  // If arg is not a valid target and no current target, ignore it
+}
+
+// If no targets specified, default to desktop
+if (
+  !targets.desktop.enabled &&
+  !targets.sandbox.enabled &&
+  !targets.sync.enabled
+) {
+  targets.desktop.enabled = true;
+}
+
+const subProcesses: child_process.ChildProcess[] = [];
+async function cleanup(signal: NodeJS.Signals) {
+  for (const p of subProcesses) p.kill(signal);
+  console.log('\nstart    | requested termination.');
+  await timers.setTimeout(5000);
+  const stillRunning = subProcesses.filter((p) => p.exitCode === null);
+  for (const p of stillRunning) p.kill('SIGKILL');
+  console.log('\nstart    | done.');
+  process.exit(0);
+}
+
+process.on('SIGINT', cleanup).on('SIGTERM', cleanup);
+
+// Helper function to create a transform stream that prefixes lines
+function createPrefixTransform(prefix: string) {
```

Comment:
Added this because it was hard to debug issues when all these processes are sharing the terminal. In general there should be no errors that aren't obvious because it would be some webpack error that has a file path in it, this helped more with the scripts throwing something. 

We can take out the prefixing or keep some of it in a simpler form and just disable it since it's only for debugging.